Blog - Hotjar for mobile (#10135)
PostHog/posthog.com: contents/blog/hotjar-for-mobile-apps-alternatives.mdx

---

> title: 'Hotjar alternatives for mobile apps (iOS & Android)'

Author: Hmm not sure I fully agree:
1. I think Google understands the different between "hotjar alternatives" and "hotjar alternatives for mobile".
2. Here are keywords I found volume for:
    - hotjar alternatives for mobile apps (40/m)
    - Hotjar for mobile apps  (210/m)
    - Hotjar for mobile (10, ahrefs)

So having "alternatives" in the title feels like it will help us rank for some riches.

In any case, if you still feel we should avoid it, here are some alternatives I came up with:
- Hotjar replacements for mobile apps (iOS & Android) – I dont really like this since "replacement" implies it already existed
- Hotjar-like tools for mobile apps (iOS & Android) – reads a bit better but Im worried it wont rank for the above searches
- Hotjar equivalents for mobile apps (iOS & Android)

> title: 'Hotjar alternatives for mobile apps (iOS & Android)'

Reviewer: Experience has taught me otherwise! You'd be amazed how dumb / annoying google can be about stuff like this. Best to play it safe. 

I've put in "What's the best Hotjar for mobile apps?" as the title, and added a meta description that mentions the frameworks.

date: 2024-12-10
author:
  - lior-neu-ner
rootpage: /blog
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/clarity-alternatives/clarity-alternatives.png
featuredImageType: full
category: General
tags:
  - Comparisons
---

import { ComparisonTable } from 'components/ComparisonTable'
import { ComparisonRow } from 'components/ComparisonTable/row'

> While [Hotjar](/blog/posthog-vs-hotjar) is an excellent tool for viewing session replays and understanding user behavior, it doesn't support mobile apps. 
> 
> In this post, we compare the best alternatives for mobile so that you can find the best replay tool for your needs.

Reviewer: ```suggestion
Hotjar is one of the most popular session replay tools on the market, but it doesn't work on iOS and Android mobile apps, or support popular app frameworks like React Native and Flutter. 

This guide compares replay tools that support mobile apps, so you can find the best replay tool for your needs. If you don't need a "Hotjar for mobile", see our guide to [Hotjar alternatives for web apps](https://posthog.com/blog/best-hotjar-alternatives).
```

Reviewer: We should add a link to this guide from the Hotjar alternatives guide with the anchor text "Hotjar for mobile". Linking between them like this should help Google understand which page to rank for our desired term and prevent internal conflict.

> This guide compares replay tools that support mobile apps, so you can find the best replay tool for your needs. If you don't need a "Hotjar for mobile", see our guide to [Hotjar alternatives for web apps](https://posthog.com/blog/best-hotjar-alternatives).

Author: ```suggestion
This guide compares replay tools that support mobile apps, so you can find the best replay tool for your needs. If you don't need a "Hotjar for mobile", see our guide to [Hotjar alternatives for web apps](/blog/best-hotjar-alternatives).
```

3. **It's free:** The short 30-day retention limit is easy to forgive when it's free.

> #### Bottom line
> 
> If you only care about session replay and dont need other features, Clarity is the obvious choice - especially if you're already using Google Analytics. On the other hand, software engineers will find Clarity's lack of debug logs disappointing.

<br />

## 4. PostHog

**Typical users** Engineering and product teams
  
![posthog](https://res.cloudinary.com/dmukukwp6/image/upload/v1722527045/posthog.com/contents/posthog-mobile-replay.png)

### What is PostHog?

[PostHog](https://posthog.com) (that's us 👋) is an all-in-one platform built to help engineers create better products. It includes [session replays](/session-replay) **and** a whole bunch more, such as [product analytics](/analytics), [A/B testing](/experiments), and [feature flags](/feature-flags).

It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.

#### Mobile replay features

<ComparisonTable column1="PostHog">

>   <ComparisonRow column1="Android, iOS" feature="Supported platforms" description="Mobile platforms where session replay is available" />

Reviewer: ```suggestion
  <ComparisonRow column1="Android, iOS, React Native, Flutter" feature="Supported platforms" description="Mobile platforms where session replay is available" />
```

We should update other alternatives and comparisons that have this data in them while we're here.

  <ComparisonRow column1={true} feature="Debug logs" description="View console logs, network logs" />
  <ComparisonRow column1={true} feature="Filters" description="View specific replays based on user segments, screens visited, events" />
  <ComparisonRow column1={false} feature="Heatmaps" description="Highlight areas that get the most interaction" />
  <ComparisonRow column1={false} feature="Scroll maps" description="See how far users scroll and where they dwell" />
  <ComparisonRow column1={false} feature="Rage taps" description="Identify repetitive taps made out of frustration" />

>   <ComparisonRow column1={false} feature="Dead taps" description="Detect taps on elements that do not respond or trigger an action" />

Reviewer: Do we 100% not support these features on mobile now? I know we support them on web SDKs. Just want to check we're not selling ourselves short here!

Author: the other features too right? Rage taps etc.

Reviewer 2: correct, but we do support https://posthog.com/docs/session-replay/network-recording (except flutter) so I'd list it next to Debug logs

  <ComparisonRow column1={false} feature="Crash reports" description="Learn what happens before a crash without complex debugging or reproduction" />
  <ComparisonRow column1={false} feature="AI summaries" description="Generate summaries of session replays" />
</ComparisonTable>

#### Other key features

- 📈 **Product analytics:** Custom trends, funnels, user paths, retention analysis, and segment user cohorts. Also, direct [SQL querying](/docs/product-analytics/sql) for power users.

- 🧪 **A/B tests:** Experiment in your app with up to nine test variations and track impact on primary and secondary metrics. Auto-calculate test duration, sample size, and statistical significance.

- 🚩 **Feature flags:** Rollout features safely with [local evaluation](/docs/feature-flags/local-evaluation) (for faster performance), JSON payloads, and instant rollbacks.

- 🏠 **Data warehouse:** Combine data from all your different sources for easy analysis and comprehensive insights.

### How much does PostHog cost?

> PostHog has [transparent pricing](/session-replay#pricing) based on usage. It’s free to get started and completely free for the first 5,000 recordings. After this free monthly allowance, pricing starts at $0.0050/recording (or $50 for 10k recordings), and recordings cost progressively less the more you use. This makes PostHog significantly cheaper than all the other companies on this list (apart from [Microsoft Clarity](#3-microsoft-clarity)).

Reviewer: ```suggestion
PostHog has [transparent pricing](/session-replay#pricing) based on usage. It’s free to get started and completely free for the first 5,000 recordings. After this free monthly allowance, pricing starts at $0.0050/recording (or $50 for 10k recordings), and recordings cost progressively less the more you use. With the exception of Microsoft Clarity, this makes PostHog significantly cheaper than all the other tools on this list.
```
